content/opt/setup/11-timezone

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+. /opt/helpers
+
+[ -n "${TZ-}" ] || exit 0
+
+echo "=> Setting timezone to ${TZ}"
+
+TZFILE="/usr/share/zoneinfo/${TZ}"
+if [ ! -f ${TZFILE} ]; then
```

Comment:
This explodes, when TZ contains a space. Please use quotes _everywhere_.

Reply:
Done.